js/ui/marker.js

```diff
@@ -5,22 +5,31 @@ module.exports = Marker;
 
 var DOM = require('../util/dom');
 var LngLat = require('../geo/lng_lat');
+var Point = require('point-geometry');
 
 /**
  * Creates a marker component
  * @class Marker
  * @param {HTMLElement=} element DOM element to use as a marker (creates a div element by default)
+ * @param {Object} options
+ * @param {Point} options.offset The offset in pixels to apply to this Marker relative to the element's top left corner.
```

Comment:
A corresponding Leaflet term is `anchor`, which would be equivalent to `-offset`. Should we align the terminology here, or is `offset` better? Not sure.